packages/cfd/src/Containers/jurisdiction-modal-content.tsx

```diff
@@ -0,0 +1,823 @@
+import React from 'react';
```

Comment:
@prince-deriv i can't see any bot-related changes in jurisdiction-modal-content.tsx or jurisdiction-modal.tsx. And as I can see, they are located in a different directory on master now. Please don't add them back inside 'Containers' again🙂